app/components/BioEditor.tsx

```diff
@@ -358,6 +389,77 @@ export const BioEditor = (props: BioEditorProps) => {
       ) : (
         ''
       )}
+      <h4>Social sharing buttons</h4>
+      {isShareButtonsOpen ? (
```

Comment:
When I'm editing the text, should we prompt to confirm on a page refresh/navigation? Seems like a small detail for a short field, but could be a pattern we want to adopt throughout?

Reply:
Another great thought. But yeah it seems like maybe something we should do everywhere as part of an overhaul or even just as a one-off task to start.